backend/app/services/langgraph_chat/tools/mcp/session.py

```
@@ -0,0 +1,126 @@
+                    server_config["url"],
+                    server_config.get("headers"),
+                )
+            else:
+                raise ValueError(f"Unknown MCP server type: {server_type}")
+
+            # Cache adapted tools
+            session = self.client.get_session(server_name)
+            if session:
+                self.tools_cache[server_name] = adapt_mcp_tools(session)
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)